webservices/schemas.py

```diff
@@ -699,6 +699,7 @@ def remove_fec_url(self, obj):
     fields={
         'beginning_image_number': ma.fields.Str(),
         'ending_image_number': ma.fields.Str(),
+        'html_url': ma.fields.Str(),
```

Comment:
There is a question about if this is the right schema, it is.   The other one that is similarly named are schemas that are used to build the efile endpoints, and shouldn't be referenced directly.  Schemas.py is a bit of a mess.